books/free-programming-books-es.md

```diff
@@ -252,9 +245,6 @@
 
 #### React
 
-* [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) - Sergio Xalambrí (PDF, EPUB, MOBI; Leanpub cuenta requerida)
```

Comment:
It's free to read online :worried:

Reply:
I'll write a script to check them all; when free to read HTML we can remove the access note.

Reply:
> ... when free to read HTML we can remove the access note.

Suggestion updated